x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/logic/diff/normalization/convert_rule_to_diffable.ts

```diff
@@ -207,8 +207,6 @@ const extractDiffableThreatMatchFieldsFromRuleObject = (
     threat_index: rule.threat_index,
     threat_mapping: rule.threat_mapping,
     threat_indicator_path: rule.threat_indicator_path,
-    concurrent_searches: rule.concurrent_searches,
```

Comment:
Do you think it would be a good idea to add a comment explaining why these two fields are not included in the `DiffableRule` object? 
Otherwise maybe one day someone will notice that they are missing and add them back.